src/services/product.services.test.ts

```diff
@@ -0,0 +1,246 @@
+import { faker } from '@faker-js/faker';
+import { db } from '../../prisma/db';
+import { ApiError } from '../config/apiError';
+import { errors } from '../config/errors';
+import { ProductSerializer } from '../serializers/product-serializer';
+import { createProductRandom, productIndexRandomRaw, productRandomRaw } from '../test/test-constants';
+import { createProduct, createUser } from '../test/utils';
+import { Product, ProductIndex, ProductIndexRaw, UpdateProductRequest } from '../types/product';
+import { ProductService } from './product.services';
+
+describe('Products Service', () => {
+  let userId: number;
+
+  beforeAll(async () => {
+    userId = (await createUser()).id;
+  });
+
+  afterEach(async () => {
+    jest.restoreAllMocks();
+  });
+
+  afterAll(async () => {
+    await db.user.delete({
+      where: { id: userId },
+    });
+  });
+
+  describe('Create Product', () => {
```

Comment:
Please test that the product is created

Reply:
Check the new file `product.services.new.test.ts`